test/regression.py

```diff
@@ -3353,12 +3366,33 @@ def testatommasses(self):
     "Psi4/Psi4-beta5/water_ccsd.out":   GenericCCTest,
     "Psi4/Psi4-beta5/water_mp2.out":    GenericMP2Test,
 
+    "QChem/QChem4.2/C_bigbasis.out":         QChemBigBasisTest,
+    "QChem/QChem4.2/MoOCl4_sp.out":          GenericCoreTest,
+    "QChem/QChem4.2/Trp_polar.out":          ReferencePolarTest,
+    "QChem/QChem4.2/dvb_bomd.out":           GenericBOMDTest,
+    "QChem/QChem4.2/dvb_gopt.out":           GenericGeoOptTest,
+    "QChem/QChem4.2/dvb_ir.out":             QChemIRTest,
+    "QChem/QChem4.2/dvb_raman.out":          QChemRamanTest,
+    # also GenericBasisTest

```

Comment:
Can you explain what these comments are referring to?

Reply:
`dvb_sp.out` is always used for at least the `Basis` and `SP` tests.  If you consider this being a dict from filename to unit test class, then you can only have one test per file.  If you look there are actually a lot of duplicated keys which will lead to skipped unit tests.